services/StudentEntryService.java

```diff
@@ -0,0 +1,13 @@
+package services;
+
+import logic.models.Student;
+
+public interface StudentEntryService {
```

Comment:
What is a `StudentEntry` in comparison to a `Student`? Maybe a better name would be `StudentService`.

Reply:
Updated.